ocd-3393.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE openchpl.certified_product
+ADD COLUMN IF NOT EXISTS rwt_plan_url text NULL;
+
+ALTER TABLE openchpl.certified_product
+ADD COLUMN IF NOT EXISTS rwt_plan_submission_date date NULL;
```

Comment:
This column is a `date` type. The value in the API is a `long` type. I think the API is wrong, but if it's correct then this is wrong